frontend/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@types/jest": "^23.3.10",
+    "@types/node": "^10.12.15",
+    "@types/react": "^16.7.17",
+    "@types/react-dom": "^16.0.11",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-scripts": "2.1.1",
+    "typescript": "^3.2.2"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "eslint 'src/**/*.tsx'"
```

We have a `format` script in gitbase-web
```suggestion
    "lint": "eslint 'src/**/*.tsx'",
    "format": "eslint --fix 'src/**/*.js'"
```

---

I decided to leave it for now because of typescript. I'm studing documentation for the language itself right now, then I'll check ecosystem. I'm not sure we need eslint with typescript at all.